hadoop-tools/hadoop-azure/src/test/java/org/apache/hadoop/fs/azurebfs/contract/ITestAbfsFileSystemContractOpen.java

```diff
@@ -49,4 +61,55 @@ protected Configuration createConfiguration() {
   protected AbstractFSContract createContract(final Configuration conf) {
     return new AbfsFileSystemContract(conf, isSecure);
   }
+
+  @Override
+  public FileSystem getFileSystem() {
```

Comment:
This code is repeated at multiple places, can it be made centralized

Reply:
+1 on that!

Have taken in AbfsFileSystemContract#getTestFileSystem. This method is used to supply fs to the contract tests.